packages/api/prisma/schema.prisma

```diff
@@ -8,9 +8,13 @@ generator client {
 }
 
 model User {
-  id       Int    @id @default(autoincrement())
-  githubId Int    @unique
-  fullName String
-  email    String
-  image    String
+  id              String  @id
+  githubId        Int     @unique
+  fullName        String?
```

Comment:
If those properties are nullable (fullName, email, town), maybe there is a candidate for a totally different module? 
It gives me a sign that it's possible that some of those properties will be usable in just certain use cases. It's all about cohesion, like in classes' design.

https://ducmanhphan.github.io/2019-03-23-Coupling-and-Cohension-in-OOP/